### DISCUSSION TITLE
一直bind unbind不干活。

### DISCUSSION THREAD

**Discussion by Bourne1314** [apache/incubator-seata#4202] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/23721287/145820143-428c89cd-2fe6-4659-9891-69c0c7eb7e70.png)
如图，一直显示bind和unbind，不知道什么原因，xid能收到。

**Comment by wangliang181230** [comment, marked as answer]
只是其他日志没打印而已吧。应该没问题。

**Reply by Bourne1314** [reply]
远程事务未回滚，只有本地事务回滚了。bind和unbind在我的demo中不会反复出现，在项目中就出现这个错误。实在费解。
入门教程反复看了很多遍，找不出问题所在。哭~

**Reply by funky-eyes** [reply]
这个不是错误，线程结束就unbin正常操作，要觉得有问题，自己排查数据源代理、xid传递、分支事务的异常是否抛到了发起方
